config/magpie.ini

```diff
@@ -64,15 +82,6 @@ cache.acl.enabled = false
 cache.service.enable = false
 # cache.service.expire = 10
 
-# ziggurat
```

Comment:
It looks like this config is now loaded dynamically. Not so sure where this config should be put as they're is no config file containing them anymore. Maybe a little message here telling admins where they should configure ziggurat from now on.

Reply:
This is now included directly in the code statically instead of dynamically by INI loaded when starting Magpie becausee a config that did not provide them simply failed to work as intended. It is something that cannot be configured. 
For someone not working actively with Magpie, it is very hard to figure out how that could be the cause of error when it was omitted by error.